src/components/SearchField.tsx

```diff
@@ -1,38 +1,221 @@
 import React, { useState } from 'react';
+import { useHistory } from 'react-router';
+
+import '../styles/SearchField.scss';
 
 const loupeIcon = '/images/search-icon.svg';
 
+const genres = [
+  'Comedy',
+  'Drama',
+  'Romance',
+  'Mystery',
+  'Thriller',
+  'Animation',
+  'Action',
+  'Adventure',
+  'Crime',
+  'Fantasy',
+  'Science Fiction',
+  'Horror',
+  'Family'
+];
+
+const checkedImg = '/images/searchFieldIcons/checked.svg';
+const uncheckedImg = '/images/searchFieldIcons/unchecked.svg';
+
 const SearchField: React.FC = () => {
+  const history = useHistory();
+
   const [inputValue, setInputValue] = useState<string>('');
 
-  const changeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+  const [focus, setFocus] = useState<boolean>(false);
+
+  const [dropdown, setDropdown] = useState<boolean>(false);
+
+  const [dateRange, setDateRange] = useState<{
+    fromYear: number;
+    toYear: number;
+  }>({
+    fromYear: 1980,
+    toYear: 2021
```

Comment:
avoid magic numbers, use const
toYear: 2021 >> new Date().getFullYear() - so no need to change this line every year 01.01

Reply:
replaced with const and new Date().getFullYear()